server/public/model/job.go

```diff
@@ -118,15 +122,19 @@ func (j *Job) IsValid() *AppError {
 	return nil
 }
 
+// InitLogger attaches an annotated logger to a Job.
+// It should always be called after creating a new Job to ensure `Job.Logger` it set.
+func (j *Job) InitLogger(logger mlog.LoggerIFace) {
+	j.Logger = logger.Sugar(
```

Comment:
further to the previous comment, this line can be replaced with `j.Logger = logger.With(...)` which leaves the job developer with the option to use more structured logging or sugar.